### DISCUSSION TITLE
Externalize "suppress what's new dialog" setting to environment variable

### DISCUSSION THREAD

**Discussion by haydonryan** [open-webui/open-webui#6463] [Ideas]
# Feature Request

---

**Describe the solution you'd like**
Could you add a user preference to suppress the  "what's new" popup.  I update nightly, and when I want to use the openbweui i just want  to get in and do my work.  I periodically check the what's new from the about field in settings and have dedicated exploration sessions.

Another option would be to show what's new on the landing page, but allow the user to start typing their next chat. 

I know it's a trivial feature request but it'd be awesome if you'd consider it.

PS: Love openwebui - keep up the great work and update cadence!

**Comment by tjbck** [comment]
I'll take a look and add it as one of the interface options!

**Reply by haydonryan** [reply]
Thank-you so much!

**Reply by silentoplayz** [reply]
> Thank-you so much!

Responding for the maintainer here, but it *should* be added to the dev branch with https://github.com/open-webui/open-webui/commit/e6ca994c92ed058865f326c8f9625cf33b9c4647 commit!
